lib/ansible/module_utils/netapp.py

```diff
@@ -245,7 +258,8 @@ class NetAppESeriesModule(object):
     """
     DEFAULT_TIMEOUT = 60
     DEFAULT_SECURE_PORT = "8443"
-    DEFAULT_REST_API_PATH = "devmgr/v2"
+    DEFAULT_REST_API_PATH = "devmgr/v2/"
+    DEFAULT_REST_API_ABOUT_PATH = "devmgr/utils/about"
```

Comment:
we should not have changes in the E-series parts.

Reply:
let me rebase (there where changes made to netapp.py the day after i made this which are in our bitbucket copy but not this copy since i haven't rebased)

Reply:
might have been me @carchi8py I just merged the changes to `netapp.py`

Reply:
No i was missing change made 19 day ago since my pull request was 20 day ago.. But i also got your changes as well when i rebased